virtool/example/paths.py

```diff
@@ -0,0 +1,3 @@
+from pathlib import Path
+
+example_path = Path(__file__).parent.parent.parent / "example"
```

Comment:
I think this should be moved to a module at `virtool/example.py`. No need to have another directory for a single constant.